docs/reference/release-notes/8.0.0-alpha1.asciidoc

```diff
@@ -18,4 +18,11 @@ Mapping::
 Deprecations::
 * Remove undocumented endpoints of hot threads API {pull}55109[#55109]
 
+Slow loggers::
+* `index.indexing.slowlog.level` and `index.indexing.slowlog.level` are removed. These settings can be worked around
```

Comment:
It looks like the settings listed here are the same? One should be the search slowlog level?

Reply:
yes, good find.